ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v1/TransactionsService.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.index.v1
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.daml.ledger.participant.state.v1.Offset
+import com.digitalasset.daml.lf.transaction.BlindingInfo
+import com.digitalasset.ledger.api.domain.TransactionFilter
+
+import scala.concurrent.Future
+
+trait TransactionsService {
```

Comment:
Just realized it would be helpful to document what these are used for. E.g. this is used to implement the transaction trees service (and transaction service for now).

Reply:
added docs around the new interfaces